sendgrid-email/addtolist_es6.js

```diff
@@ -4,8 +4,8 @@ module.exports = (event) => {
 
   return new Promise((resolve, reject) => {
     var request = sg.emptyRequest()
-    const { email, first_name: nameFirst, last_name: nameLast } = event.data.User.node
-    request.body = [{email, first_name, last_name }];
+    const { email, nameFirst, nameLast } = event.data.User.node
+    request.body = [{email, first_name: nameFirst, last_name: nameLast }];
```

Comment:
Can you try `request.body = [({ email, first_name, last_name } = event.data.User.node)]`?